### PR TITLE
Add custom scrolling element

### PR DESCRIPTION
**Pull request by jonasCr**
After reading #385 issue, I implements the feature by myself.
I would be happy to help this project with my first ever contribution to open source ;) 

Any comments is fully appreciate 👍

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -22,6 +24,7 @@ export default ({
       // Needs to be a valid DOM Element
       if (!section || !(section instanceof Element)) continue;
       // GetBoundingClientRect returns values relative to viewport
+      section.addEventListener
```
Comment (Purii) [inline src/index.ts:27]: Is that line added by accident?

File: src/index.ts [hunk 2]
```diff
@@ -34,13 +37,17 @@ export default ({
   });
 
   useEffect(() => {
-    window.addEventListener('scroll', handle);
+    let scrollable = scrollingElement == null
```
Comment (Purii) [inline src/index.ts:40]: No need to compare to `null`.
`scrollingElement?.current ? scrollingElement.current : window ` should be sufficient

**Review by Purii — CHANGES_REQUESTED** [review]
Hi @jonasCr !
Thanks for the PR. That feature makes sense!
I added two comments :)

File: src/index.ts (hunk 1 above)
Comment (jonasCr) [reply src/index.ts:27]: Yep, I will fix it in the next commit

File: src/index.ts (hunk 2 above)
Comment (jonasCr) [reply src/index.ts:40]: OK, changing it....

**Comment by jonasCr** [comment]
Requested change applied :)

**Pull request merged**